packages/account/src/Sections/Profile/LanguageSettings/language-settings.tsx

```diff
@@ -25,13 +25,9 @@ const LanguageSettings = observer(() => {
         switchLanguage(language_key);
     };
 
-    let allowed_languages: Record<string, string> = getAllowedLanguages(UNSUPPORTED_LANGUAGES);
-
-    if (has_wallet) {
-        allowed_languages = Object.fromEntries(
-            Object.entries(allowed_languages).filter(([language_key]) => ['EN', 'AR'].includes(language_key))
-        );
-    }
+    const allowed_languages: Record<string, string> = getAllowedLanguages(
+        has_wallet ? WALLETS_UNSUPPORTED_LANGUAGES : UNSUPPORTED_LANGUAGES
```

Comment:
Generally if a language is not supported in legacy it shouldn't be supported in wallets as well, so I've updated WALLETS_UNSUPPORTED_LANGUAGES to extend UNSUPPORTED_LANGUAGES instead to avoid duplication 👍